libstuff/SRandom.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include <random>
+using namespace std;
+
+// Random number generator class.
+class SRandom {
+private:
+    static mt19937_64* generator;
```

Comment:
Please prefix private attributes with an underscore, to be consistent with the rest of the codebase.